routes/models/proposal.js

```diff
@@ -0,0 +1,132 @@
+var md5= require('md5') ;
+var DB = require('../config.js') ;
+var random = require('../../libs/random.js') ;
+var Boom =  require('boom');
+
+module.exports.add = function(req,res) {
+    var proposal = req.payload ;
+    proposal.created =  new Date();
+    proposal.is_active = 1;
+    DB.conn.queryAsync("INSERT INTO "+DB.proposals +" SET ?" , proposal ).then(function(result)
+    {
+        res({"statusCode":200,"message":"New proposal created successfully"}) ;
+    });
+}
+
+module.exports.proposals = function(req,res) {
+
+    var total_count = 0 ;
+
+    if(!req.query.page)
+    {
+        res(Boom.badRequest('Please provide the page number'));
+    }
+
+    if(req.query.page <= 0)
+    {
+        res(Boom.badRequest('Invalid page number.'));
+    }
+
+    var prStatus = req.params.status ;
+    var perpage =  1;
+    var page = req.query.page ;
+    var search = false   ;
+    if(req.query.s) {
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic there should be specific names for parameters. I am not getting what do you mean by s